testing_and_setup/compass/ocean/global_ocean/config_files_ISC/config_initial_state.xml

```diff
@@ -53,6 +56,12 @@
 	</streams>
 
 	<run_script name="run.py">
+		<step executable="./smooth_topo.py">
+			<argument flag="--iter">100</argument>
+			<argument flag="--smoothing">1.0</argument>
+			<argument flag="--mpi_tasks">4</argument>
```

Comment:
It's annoying that the number of MPI tasks is repeatedly hard coded and not part of the config file or some kind of machine config file.  We need to think if there's a better way to do this.  This is just one example of where the problem arises.